### PR TITLE
VRF get-handler

### PR DESCRIPTION
**Pull request by GarrickHe**
* Update: VRF get-handler support
* Update: VRF configuration script

Signed-off-by: Garrick He <garrick_he@dell.com>

### CODE REVIEW

File: scripts/bin/opx-config-vrf [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    print '-c, --create      : Add VRF to the configuration'
+    print '-d, --delete      : Delete the specific VRF'
+    print '-v, --vrf         : VRF name that user wants to create'
+    print '-p, --port        : port(s) that need to be added for a VRF'
+    print '--addport         : option to add port to the given VRF'
+    print '--removeport      : option to delete port from the given VRF'
+    print '-s, --show        : show the VRF parameter, when no VRF name given show all'
+
+    print 'Example:'
+    print 'os10-config-vrf  --create --vrf blue --port e101-001-0,e101-004-0'
```
Comment (rakeshdatta) [inline scripts/bin/opx-config-vrf]: Modify to opx-config-vrf

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+opx-nas-interface (5.18.0+opx6) unstable; urgency=medium
+
+  * Update: VRF get-handler support
+  * Update: VRF configuration script
+
```
Comment (rakeshdatta) [inline debian/changelog]: plz mention about t he bugfix for vrf creation failure

**Comment by GarrickHe** [comment]
Unit-test:

root@OPX:~# opx-config-vrf --create --vrf blue
Configuration Successful...
root@OPX:~# opx-config-vrf --show
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = blue
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = default
root@OPX:~# opx-config-vrf --create --vrf red
Configuration Successful...
root@OPX:~# opx-config-vrf --show
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = red
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = blue
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = default
root@OPX:~#
root@OPX:~# opx-config-vrf  --create --vrf green --port e101-001-0,e101-004-0
Configuration Successful...
Configuration Successful...
Configuration Successful...
root@OPX:~# opx-config-vrf --show
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = green
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = red
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = blue
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = default
root@OPX:~# opx-config-vrf  --delete --vrf blue
Configuration Successful...
root@OPX:~# opx-config-vrf --show
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = green
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = red
Key: 1.292.44.2883617.2883612.2883613.
ni/network-instances/network-instance/name = default
root@OPX:~#
root@OPX:~#
root@OPX:~#
root@OPX:~# cps_set_oid.py -oper create ni/network-instances ni/network-instances/network-instance/name="management"
Success
Key: 1.292.19136521.
ni/network-instances/network-instance/name = management
cps/object-group/return-code = 0
root@OPX:~# cps_set_oid.py -oper create ni/network-instances ni/network-instances/network-instance/name="data1"
Success
Key: 1.292.19136521.
ni/network-instances/network-instance/name = data1
cps/object-group/return-code = 0
root@OPX:~#
root@OPX:~#
root@OPX:~# cps_get_oid.py -qua target ni/network-instances

============1.292.44.2883617.2883612.2883613.==========
ni/network-instances/network-instance/name = management
------------------------------------------------
ni/network-instances/network-instance/name = green
------------------------------------------------
ni/network-instances/network-instance/name = red
------------------------------------------------
ni/network-instances/network-instance/name = default
------------------------------------------------
root@OPX:~# cps_get_oid.py -qua target ni/network-instances ni/network-instances/network-instance/name="management"

============1.292.44.2883617.2883612.2883613.==========
ni/network-instances/network-instance/name = management
------------------------------------------------
root@OPX:~# cps_get_oid.py -qua target ni/network-instances ni/network-instances/network-instance/name="data1"

============1.292.44.2883617.2883612.2883613.==========
ni/network-instances/network-instance/name = data1
------------------------------------------------
root@OPX:~#

File: scripts/bin/opx-config-vrf (hunk 1 above)
Comment (GarrickHe) [reply scripts/bin/opx-config-vrf]: will do.

File: debian/changelog (hunk 2 above)
Comment (GarrickHe) [reply debian/changelog]: will do

**Comment by GarrickHe** [comment]
Pull-request updated, HEAD is now b10963ab769f47111acde4465bcc2e81f6be9a34

**Pull request merged**